src/universal/components/CreateCard/CreateCard.js

```diff
@@ -80,18 +69,41 @@ CreateCard.propTypes = {
   styles: PropTypes.object
 };
 
-const labelStyles = {
-  display: 'inline-block',
+const borderRadius = '.1875rem';
```

Comment:
I know what you mean. Here is what I had in mind:

http://recordit.co/y4onoyGUey

I’ve (mostly) used the `rem` unit, so changing the `16px` on the `html` selector is mostly like a zoom. Margins, paddings, border-radius, font-size all change based on that base multiplier.

What we could do, and would be interesting when we start exploring adaptive/responsive/mo-mobile-first, is use something fixed for “UI chrome” elements (certain panels, borders, etc.) and use `rem` for text and elements that need the proper sizing/spacing. Basically the chrome wouldn’t change but the other elements would get larger/smaller. This is all based on the idea of letting the user set their default browser font-size. We would remove the font-size from html/body.